File: lessonOne/integration.js
// Example of making a GET request using Fetch API in JavaScript

// URL of the API endpoint
const apiUrl = 'https://jsonplaceholder.typicode.com/posts/1';

// Making a GET request
fetch(apiUrl)
    .then(response => {
        // Check if the response is successful
        if (!response.ok) {
            throw new Error('Network response was not ok ' + response.statusText);
        }
        return response.json(); // Parsing the JSON response
    })
    .then(data => {
        // Handling the data from the response
        console.log('Data received:', data);
    })
    .catch(error => {
        // Handling errors
        console.error('There was a problem with the fetch operation:', error);
    });

File: lessonOne/integration.py
# Example of making a GET request using Requests library in Python

import requests

# URL of the API endpoint
api_url = 'https://jsonplaceholder.typicode.com/posts/1'

try:
    # Making a GET request
    response = requests.get(api_url)
    # Check if the response is successful
    response.raise_for_status()
    # Parsing the JSON response
    data = response.json()
    # Handling the data from the response
    print('Data received:', data)
except requests.exceptions.HTTPError as http_err:
    # Handling HTTP errors
    print(f'HTTP error occurred: {http_err}')
except Exception as err:
    # Handling other errors
    print(f'Other error occurred: {err}')